python/cudf/cudf/tests/test_rolling.py

```diff
@@ -47,15 +47,7 @@ def test_rollling_series_basic(data, index, agg, nulls, center):
             got = getattr(
                 gsr.rolling(window_size, min_periods, center), agg
             )().fillna(-1)
-            try:
```

Comment:
Please fix the spelling of the function name (can't comment on it, but it is `test_rollling_series_basic` (3 letter `l`).

Reply:
Yes, that bothered me. Will fix.